Prettier doesn't format *.css and  *.scss files.

Here are my settings: 

```
"prettier.eslintIntegration": true,
"prettier.singleQuote": true,
"prettier.printWidth": 80,
"prettier.trailingComma": "all",
"prettier.cssEnable": [
    "css",
    "less",
    "sass"
],
```

But there is no "Format document" option in the content menu in vscode for *.css and *.scss files.

Prettier version: [0.19.0]

What I am doing wrong? Prettier formats .js files fine.

I've checked Developer Tools console:

```
extensionHost.ts:285[Extension Host] Activating extension `esbenp.prettier-vscode` failed:  Cannot find module '/Users/serge/.vscode/extensions/esbenp.prettier-vscode-0.19.0/out/src/extension'
d.logExtensionHostMessage @ extensionHost.ts:285
extensionHost.ts:285[Extension Host] Here is the error stack:  Error: Cannot find module '/Users/serge/.vscode/extensions/esbenp.prettier-vscode-0.19.0/out/src/extension'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
```

So I think that problem with installation of the latest version.

--- a/README.md
+++ b/README.md
@@ -77,7 +77,7 @@ Advanced feature. Use this to opt in / out prettier on various language ids. Res
 Use parser `typescript` for given language ids.
 Use with care.
 
-#### prettier.cssEnable (default: ["css", "less", "sass"])
+#### prettier.cssEnable (default: ["css", "less", "scss"])
 Advanced feature. Use this to opt in / out prettier on various language ids. Restart required.
 Use parser `postcss` for given language ids.
 Use with care.

--- a/package.json
+++ b/package.json
@@ -151,7 +151,7 @@
                 "enum": [
                   "css",
                   "less",
-                  "sass"
+                  "scss"
                 ]
               }
             ]
@@ -159,7 +159,7 @@
           "default": [
             "css",
             "less",
-            "sass"
+            "scss"
           ]
         }
       }

--- a/src/types.d.ts
+++ b/src/types.d.ts
@@ -23,7 +23,7 @@ export interface PrettierConfig {
 interface Language {
     javascript: ('javascript' | 'javascriptreact' | string)[];
     typescript: ('typescript' | 'typescriptreact' | string)[];
-    css: ('css' | 'less' | 'sass' | string)[];
+    css: ('css' | 'less' | 'scss' | string)[];
 }
 /**
  * prettier-vscode specific configuration
@@ -45,7 +45,7 @@ interface ExtensionConfig {
     /**
      * Language ids to run postcss prettier on.
      */
-    cssEnable: ('css' | 'less' | 'sass' | string)[];
+    cssEnable: ('css' | 'less' | 'scss' | string)[];
 }
 
 /**

